README.md

```diff
@@ -40,9 +40,7 @@ The process documents and data are made available under a CC-BY license. Softwar
 
 ## Acknowledgements
 
-The RDMkit is supported by:
-- ELIXIR-CONVERGE https://elixir-europe.org/about-us/how-funded/eu-projects/converge
-- Contributions from the 23 ELIXIR Nodes
+The RDMkit was supported by [ELIXIR-CONVERGE](https://elixir-europe.org/about-us/how-funded/eu-projects/converge).
```

Comment:
The RDMkit was supported by [ELIXIR-CONVERGE](https://elixir-europe.org/about-us/how-funded/eu-projects/converge) and it is coordinated by ELIXIR Europe.

Reply:
Done!

Reply:
Done!